docs/user/rstudio/ide/guide/pkg-devel/writing-packages.qmd

```diff
@@ -2,33 +2,37 @@
 title: "Writing Packages"
 ---
 
-## Overview
-
-Adapted from <https://support.rstudio.com/hc/en-us/articles/200486488-Developing-Packages-with-RStudio>
-
-R packages are an ideal way to package and distribute R code and data for re-use by others. The RStudio IDE includes a variety of tools that make developing R packages easier and more productive, including:
+R packages are an ideal way to package and distribute R code and data for re-use by others. RStudio includes a variety of tools that make developing R packages easier and more productive, including:
 
 -   Build pane with package development commands and a view of build output and errors
 
 -   Build and Reload command that rebuilds the package and reloads it in a fresh R session
 
 -   R documentation tools including previewing, spell-checking, and [Roxygen](https://roxygen2.r-lib.org/) aware editing.
 
--   Integration with [devtools](https://github.com/hadley/devtools) package development functions
+-   Integration with [devtools](https://github.com/r-lib/devtools) package development functions
 
 -   Support for [Rcpp](http://dirk.eddelbuettel.com/code/rcpp.html) including syntax highlighting for C/C++ and gcc error navigation
 
 ## Getting Started
 
-Also see: Adapted from: <https://r-pkgs.org/Workflow101.html>
+The following documentation covers fundamental R package development workflows inside RStudio. The [R packages book](https://r-pkgs.org/) provides a deeper overview of general R package development strategies.
+
+::: {.callout-tip}
+
+The [**Command Palette**](../ui/command-palette.html), accessed by <kbd>Ctrl+Shift+P</kbd>, can also execute `devtools`, `usethis`, or Package Development specific commands:
+
+![](images/command-palette-pkgs.png){fig-alt="The Command Palette filtered down to package development specific commands."}
+
+:::
 
 ### Package creation
 
 Once you've come up with a name, there are two ways to create the package.
 
 -   Call [`usethis::create_package()`](https://usethis.r-lib.org/reference/create_package.html).
 
--   In RStudio, do *File \> New Project \> New Directory \> R Package*. This ultimately calls [`usethis::create_package()`](https://usethis.r-lib.org/reference/create_package.html), so really there's just one way.
+-   In RStudio, do **File** \> **New Project** \> **New Directory** \> **R Package**. This ultimately calls [`usethis::create_package()`](https://usethis.r-lib.org/reference/create_package.html), so really there's just one way.
 
 ### Package Development Basics
 
```

Comment:
L43: link redirects, but can be updated to https://r-pkgs.org/ directly

Reply:
Resolved, thanks!